src/lib/components/home/artwork_rails/artwork_rail.tsx

```diff
@@ -134,7 +134,7 @@ export class ArtworkRail extends React.Component<Props & RelayPropsWorkaround, S
     return this.props.rail.results.length > (isPad ? 3 : 6)
   }
 
-  hasStartedFetching() {
+  hasFetchedData() {
     return this.props.relay && this.props.relay.variables.fetchContent
```

Comment:
`fetchContent` is set to `true` once Relay _starts_ fetching, so this is not properly representing the method’s intention. It might be better to just add something like `didPerformFetch: true` to the state from [Relay’s ready callback](https://github.com/artsy/emission/pull/537/files/86e765319701e2314440f5d02686dcb29bd1832b..5071b665be8d2a9337689d508497c223668e3077#diff-7f065e3a7e3e00efbbb83271ba6fcf71R66) and check for that.